github3/pulls.py

```diff
@@ -313,14 +313,16 @@ def issue_comments(self, number=-1, etag=None):
         return self._iter(int(number), url, IssueComment, etag=etag)
 
     @requires_auth
-    def merge(self, commit_message='', sha=None):
+    def merge(self, commit_message='', squash=False, sha=None):
```

Comment:
done

On Thu, May 5, 2016 at 1:21 PM, Ian Cordasco <notifications@github.com>
wrote:

> In github3/pulls.py
> <https://github.com/sigmavirus24/github3.py/pull/608#discussion_r62220756>
> :
>
> > @@ -313,14 +313,16 @@ def issue_comments(self, number=-1, etag=None):
> >          return self._iter(int(number), url, IssueComment, etag=etag)
> >
> >      @requires_auth
> > -    def merge(self, commit_message='', sha=None):
> > +    def merge(self, commit_message='', squash=False, sha=None):
>
> Please move squash to the end. Since sha was there first, someone might
> be doing:
>
> pr.merge(my_message, sha)
>
> So squash should come last otherwise we will severely break existing code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sigmavirus24/github3.py/pull/608/files/6b9ef38131c0b535051c41131f04c5db575a8291#r62220756>
>